apps/vehicles/lib/vehicles.ex

```diff
@@ -12,9 +12,13 @@ defmodule Vehicles do
 
   defp children do
     streams =
-      "USE_SERVER_SENT_EVENTS"
-      |> System.get_env()
-      |> stream_children()
+      if Application.get_env(:elixir, :start_data_processes) do
```

Comment:
this was added in a prior PR that's since been merged, but will note this for a future tech debt task (sorry for the mess but thanks for the feedback!)